run.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+
+GREEN='\e[32m'
+RED='\e[0;31m'
+CLEAR='\e[0m'
+
+ColorGreen(){
+	echo -ne "$GREEN$1$CLEAR"
+}
+
+IMAGE_TAG="dev-kf-lambda-member-update"
+
+build_image() {
+  echo "==> building image: ${IMAGE_TAG}..."
+  docker build -t "$IMAGE_TAG" . -f Dockerfile-dev
+}
+
+remove_image() {
+  docker rmi "$IMAGE_TAG"
+}
+
+menu(){
+echo -ne "
+$(ColorGreen '1)') Run unit tests
+$(ColorGreen '2)') Run integration tests (make sure your local elasticsearch cluster is up)
+$(ColorGreen '3)') Run terminal (with es-net network; sharing project files)
+$(ColorGreen '4)') Run terminal (no network; sharing project files)
+$(ColorGreen '5)') Remove image
+$(ColorGreen '0)') Exit
+Choose an option: "
+        read -r a
+        case $a in
+	        1) build_image && docker run --rm -it --workdir /code "$IMAGE_TAG" /bin/bash -c "pytest tests/unit ; exit" ;;
+	        2) build_image && docker run --rm -it --network=es-net --workdir /code "$IMAGE_TAG" /bin/bash -c "pytest tests/integration ; exit" ;;
+	        3) build_image && docker run --rm -it -v "$PWD":/code --network=es-net --workdir /code "$IMAGE_TAG" /bin/bash ;;
```

Comment:
I m not too sure... I added this option `-v "$PWD":/code` so that when we are in the docker's terminal we can change source code and re-launch actions. However, it can pollute our directory with hidden python stuff (caching for instance). If someone has a better idea...